azure-pipelines.yml

```diff
@@ -0,0 +1,38 @@
+# Starter pipeline
```

Comment:
Are we using Azure Pipelines as well as Travis for builds, or was this just leftover from a test build?

Reply:
I don't think that we are using azure-pipelines, as this file was the part of PnP-Cli-publicpreview repo.

Reply:
Yes, we were using Azure Pipelines for private preview.

Reply:
Should be deleted if we are not making changes to the CI engine for this release.

Reply:
deleted